cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.md

```diff
@@ -0,0 +1,44 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024. The changes were deployed first in the [CERN Open Data QA instance](https://opendata-qa.cern.ch), and then propagated to the production instance.
```

Comment:
We should not advertise the QA instance address to the world from the PROD instance. Recall the preparatory JADE work, CMS 2016 work happening on QA. Moreover, users of the open data portal don't really care much whether this was deployed somewhere beforehand, they care more about which changes this upgrade brings, so I would focus on that and remove this phrase.